jenkins/deployment/k8s_deployment/cortx-cluster.groovy

```diff
@@ -95,28 +81,28 @@ pipeline {
             script {
                 echo "${env.cortxCluster_status}"
                 if ( "${env.cortxCluster_status}" == "SUCCESS") {
-                    MESSAGE = "K8s Build#${build_id} 3node Deployment Deployment=Passed"
+                    MESSAGE = "K8s Post Merge Build#${build_id} ${env.numberofnodes}Node Deployment Deployment=Passed"
                     ICON = "accept.gif"
                     STATUS = "SUCCESS"
                     env.deployment_result = "SUCCESS"
                     currentBuild.result = "SUCCESS"
                 } else if ( "${env.cortxCluster_status}" == "FAILURE") {
                     manager.buildFailure()
-                    MESSAGE = "K8s Build#${build_id} 3node Deployment Deployment=failed"
+                    MESSAGE = "K8s Post Merge Build#${build_id} ${env.numberofnodes}Node Deployment Deployment=failed"
                     ICON = "error.gif"
                     STATUS = "FAILURE"
                     env.deployment_result = "FAILURE"
                     currentBuild.result = "FAILURE"
                 } else {
-                    MESSAGE = "K8s Build#${build_id} 3node Deployment Deployment=unstable"
+                    MESSAGE = "K8s Post Merge Build#${build_id} ${env.numberofnodes}Node Deployment Deployment=unstable"
                     ICON = "unstable.gif"
                     STATUS = "UNSTABLE"
                     env.deployment_result = "UNSTABLE"
                     currentBuild.result = "UNSTABLE"
                 }
                 env.build_setupcortx_url = sh( script: "echo ${env.cortxcluster_build_url}/artifact/artifacts/cortx-cluster-status.txt", returnStdout: true)
                 env.host = "${env.allhost}"
-                env.build_id = "${env.dockerimage_id}"
+                env.build_id = "${CORTX_IMAGE}"
```

Comment:
build id would be `CORTX_IMAGE`? and from where `env.dockerimage_id` variable is coming which we replaced with `CORTX_IMAGE`?

Reply:
`build-id` was referred for BUILD_URL earlier so instead of creating a new email template we reused it with only assigning `CORTX_IMAGE` as a value instead of BUILD_URL. 
`CORTX_IMAGE` is input parameter as per - https://github.com/Seagate/cortx-re/blob/main/jenkins/deployment/k8s_deployment/cortx-cluster.groovy#L23 
`env.dockerimage_id` - This was borrowed from Nightly Build and it was empty for post-merge jobs.